reproducibility_project/src/utils/trajectory_conversion.py

```diff
@@ -0,0 +1,74 @@
+"""Convert Cassandra trajectories to gsd."""
+import io
+from pathlib import Path
+
+import gsd.hoomd
+import mbuild as mb
+import numpy as np
+import parmed
+
+
+def cassandra2gsd(Hpath, xyzpath, gsdpath, species_list):
+    """Convert Cassandra H and xyz files to a gsd trajectory file."""
+    Hpath = Path(Hpath)
```

Comment:
The convention in python is usually to use snake_case for class instances.
```suggestion
    h_path = Path(h_path)
```
it's fine to ignore this suggestion if you have strong feelings about it.

Reply:
I fixed it, I think.